ui/perfherder/graphs/LegendCard.jsx

```diff
@@ -13,6 +13,7 @@ const LegendCard = ({
   updateState,
   updateStateParams,
   selectedDataPoint,
+  frameworks,
   colors,
 }) => {
```

Comment:
Please provide some basic coverage, to make sure the framework name is displayed.